libcontainer/console_linux.go

```diff
@@ -38,7 +38,10 @@ func dupStdio(slavePath string) error {
 		}
 	}
 	for _, i := range []int{0, 1, 2} {
-		if err := unix.Dup3(fd, i, 0); err != nil {
+		err := utils.RetryOnEINTR(func() error {
+			return unix.Dup3(fd, i, 0)
```

Comment:
I _guess_ that dup3 can't return `EINTR`.

Reply:
My manpage here says:

```
       EINTR  The dup2() or dup3() call was interrupted by a signal; see signal(7).
```

Reply:
Yup, my guess was totally wrong, thanks for checking it!